shell/common/node_bindings.cc

```diff
@@ -188,16 +189,26 @@ void ErrorMessageListener(v8::Local<v8::Message> message,
   }
 }
 
+const std::unordered_set<base::StringPiece, base::StringPieceHash>
+GetAllowedDebugOptions() {
+  if (electron::fuses::IsNodeCliInspectEnabled()) {
+    // Only allow DebugOptions in non-ELECTRON_RUN_AS_NODE mode
```

Comment:
Is there a more robust way of disabling this functionality? I don't want to have to constantly monitor this flag list for possible changes upstream.

Reply:
This list already existed, the flag makes the list empty thus ensuring **no flags are allowed**.  This is an allow list not a block list

Reply:
Hm, okay. I know it already existed, but was hoping to make the fuse more robust. If it's an allowlist though, that covers my concern.